test/service/test_app_logging.py

```diff
@@ -66,8 +68,8 @@ def test_request_logs(self):
                             'Returning 200 response with headers {"Access-Control-Allow-Origin": '
                             '"*", "Access-Control-Allow-Headers": '
                             '"Authorization,Content-Type,X-Amz-Date,X-Amz-Security-Token,X-Api-Key", '
-                            "\"Content-Security-Policy\": \"default-src 'self';img-src 'self' data:;"
-                            "script-src 'self';style-src 'self';frame-ancestors 'none'\", "
+                            f'"Content-Security-Policy": "default-src {self_};img-src {self_} data:;'
+                            f'script-src {self_};style-src {self_};frame-ancestors {none}", '
```

Comment:
Just split the literal into two parts one with the interior double quotes and another one with interior single quotes. Use the opposite quote to delimit the literal.